common/bigquery/Makefile

```diff
@@ -102,30 +148,47 @@ storage-remove:
 	fi
 
 dataset-create:
-	$(BQ) show $(BQ_DATASET) 2>/dev/null 1>/dev/null || \
-		$(BQ) mk -d --description "$(BQ_DATASET) module" -label $(BQ_MODULE_CORE_LABEL) $(BQ_DATASET)
+	$(BQ) show $(BQ_DEPLOY_DATASET) 2>/dev/null 1>/dev/null || \
+		$(BQ) mk -d --description "$(BQ_DEPLOY_DATASET) module" -label $(BQ_MODULE_LABEL) $(BQ_DEPLOY_DATASET)
+ifdef BQ_CREATE_SPATIAL_CATALOG
```

Comment:
We could we replace this by a more generic mechanism, like a `BQ_POST_DATASET_CREATE` pointing to a shell script that we define in the data module, then we do something like this here:
```
ifdef BQ_POST_DATASET_CREATE
  BQ_POST_DATASET_CREATE
endif
```

Reply:
Good point. I have checked that a strategy using a Makefile rule `post-dataset-create:` works